samples/csharp_dotnetcore/06.using-cards/README.md

```diff
@@ -7,13 +7,27 @@ This bot has been created using [Bot Framework](https://dev.botframework.com), i
 ## To try this sample
 
 - Clone the repository
-```bash
-git clone https://github.com/Microsoft/botbuilder-samples.git
-```
-- In a terminal, navigate to `botbuilder-samples\samples\python\02.echo-bot` folder
-- Activate your desired virtual environment
-- In the terminal, type `pip install -r requirements.txt`
-- Run your bot with `python app.py`
+
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-samples.git
+    ```
+
+- Run the bot from a terminal or from Visual Studio:
+
+  A) From a terminal, navigate to `samples/csharp_dotnetcore/06.using-cards`
+
+  ```bash
+  # run the bot
+  dotnet run
+  ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/csharp_dotnetcore/11.qnamaker` folder
+  - Select `QnABot.csproj` file
```

Comment:
Can you fix these two lines to reference `06.using-cards` and `CardsBot.csproj`?